test/spec/Garn/OptparseSpec.hs

```diff
@@ -74,6 +74,32 @@ spec = around_ (hSilence [stderr]) $ do
         command <- testWithGarnTs ["run", "project", "more", "args"] ("project" ~> targetConfig)
         command `shouldBe` Run (CommandOptions "project" targetConfig) ["more", "args"]
 
+      it "parses run commands with additional flags (starting with `-`)" $ do
```

Comment:
So you need — only for —help to be parsed by the inner command? That feels unintuitive to me. Should we maybe require it for all arguments? I think not needing it made sense when we didn’t have the past command (eg “foo” in “garn build foo”) be optional, but doesn’t anymore.